src/components/ImportParameters.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+const parameterDetails = {
+  resource: { 
+    name: <><code>resource</code></>, 
```

Comment:
This mixes presentation with the data, making this unsuitable for potential reuse.